java/ql/lib/change-notes/2022-08-22-path-summaries.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* Added new flow steps for the classes `java.io.Path` and `java.nio.Paths`.
```

Comment:
It looks like the class names are incorrect here:
```suggestion
* Added new flow steps for the classes `java.nio.file.Path` and `java.nio.file.Paths`.
```
Should this be corrected in the release notes?

Reply:
Thanks @Marcono1234, that was indeed a mistake. I opened https://github.com/github/codeql/pull/10437 to address this.